client/ClientDecisionService/packages.config

```diff
@@ -18,7 +18,8 @@
   <package id="Rx-Main" version="2.2.5" targetFramework="net45" />
   <package id="Rx-PlatformServices" version="2.2.5" targetFramework="net45" />
   <package id="System.Spatial" version="5.8.1" targetFramework="net45" />
-  <package id="VowpalWabbit" version="8.4.0.3" targetFramework="net452" />
-  <package id="VowpalWabbit.JSON" version="8.4.0.3" targetFramework="net452" />
   <package id="WindowsAzure.Storage" version="7.2.1" targetFramework="net45" />
+
+  <package id="VowpalWabbit" version="8.6.0.1" targetFramework="net452" />

```

Comment:
>8.6.0.1 [](start = 38, length = 7)

Nuget.org shows 8.4.0.3

Reply:
Ah, yeah, I should not include this until we can ship a new package publicly.